core/es_isolate.rs

```diff
@@ -80,25 +82,10 @@ impl DerefMut for EsIsolate {
   }
 }
 
+// TODO(ry): a V8 Isolate cannot actually be moved between threads without the
+// use of a Locker, therefore EsIsolate should not implement the `Send` trait.
 unsafe impl Send for EsIsolate {}
```

Comment:
I agree and I'll take care of this. (I read the `TODO(username)` not so much as it's a TODO for that person, but just a note that they're the one who wrote it.)

Reply:
Oh sorry, I didn't actually mean to assign it to you. It was just copypasta from another TODO(ry) comment that I edited (https://github.com/denoland/deno/pull/4119/files#diff-7170d073ffb6b76573055a7aaba20d7fR185).